### PR TITLE
feature: add url exclude filter.

### PR DESCRIPTION
**Pull request by HoldDie**
fix: 
The browser automatically accesses the /favicon.ico address. 
and cause background error: can not match selector data: divide

add url exclude filter.

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-web/src/main/java/org/dromara/soul/web/configuration/SoulConfiguration.java [hunk 1]
```diff
@@ -161,6 +162,17 @@ public WebFilter fileSizeFilter(final SoulConfig soulConfig) {
...
     }
 
+    /**
+     * Rule out the url Filter.
+     *
+     * @return the web filter
+     */
+    @Bean
+    @Order(-5)
+    public WebFilter excludeFilter() {
```
Comment (yu199195) [inline soul-web/src/main/java/org/dromara/soul/web/configuration/SoulConfiguration.java:172]: The configuration is not enabled by default

File: soul-web/src/main/java/org/dromara/soul/web/filter/ExcludeFilter.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+/**
+ * exclude url filter.
+ *
+ * @author HoldDie
+ */
+public class ExcludeFilter implements WebFilter {
+
+    private static final AntPathMatcher MATCHER = new AntPathMatcher();
+
+    private static final Set<String> EXCLUDED_PATHS = Collections.unmodifiableSet(new HashSet<>(
```
Comment (yu199195) [inline soul-web/src/main/java/org/dromara/soul/web/filter/ExcludeFilter.java:43]: The configuration  by yml in soul-bootstrap

File: soul-web/src/main/java/org/dromara/soul/web/configuration/SoulConfiguration.java (hunk 1 above)
Comment (HoldDie) [reply soul-web/src/main/java/org/dromara/soul/web/configuration/SoulConfiguration.java]: down.

File: soul-web/src/main/java/org/dromara/soul/web/filter/ExcludeFilter.java (hunk 2 above)
Comment (HoldDie) [reply soul-web/src/main/java/org/dromara/soul/web/filter/ExcludeFilter.java]: down.

**Review by dengliming — APPROVED** [review]

**Pull request merged**